cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -182,16 +186,16 @@ private CruiseControl createCruiseControl(Kafka kafkaAssembly) {
     private Kafka createKafka(CruiseControlSpec cruiseControlSpec) {
         return new KafkaBuilder(ResourceUtils.createKafka(namespace, cluster, replicas, image, healthDelay, healthTimeout))
                 .editSpec()
-                    .editKafka()
-                        .withConfig(kafkaConfig)
-                        .withStorage(kafkaStorage)
-                   .endKafka()
-                  .withCruiseControl(cruiseControlSpec)
+                .editKafka()
+                .withConfig(kafkaConfig)
+                .withStorage(kafkaStorage)
+                .endKafka()
+                .withCruiseControl(cruiseControlSpec)
```

Comment:
Same as above and everywhere else in here.

Reply:
Indentation fixed